manual/examples/lights-ambient.html

```diff
@@ -131,7 +131,7 @@
 
 		const gui = new GUI();
 		gui.addColor( new ColorGUIHelper( light, 'color' ), 'value' ).name( 'color' );
-		gui.add( light, 'intensity', 0, 5, 0.01 );
+		gui.add( light, 'intensity', 0, 2, 0.01 );
```

Comment:
A max intensity value of `2` is too low for an ambient light.

Reply:
Yes! I didn't consider this point, the code has been updated.